tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,778 @@
+            "path": "/ACL_RULE/DYNAMIC_ACL_TABLE|RULE_2",
+            "value": { }
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+
+        expect_acl_rule_removed(duthost, "RULE_1")
+        expect_acl_rule_removed(duthost, "RULE_2")
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+def dynamic_acl_remove_table(duthost):
+    """Remove an ACL Table Type from the duthost"""
+    json_patch = [
+        {
+            "op": "remove",
+            "path": "/ACL_TABLE/DYNAMIC_ACL_TABLE",
+            "value": { }
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+def dynamic_acl_remove_nonexistant_table(duthost):
+    """Remove a nonexistent ACL Table from the duthost, verify it fails"""
+    json_patch = [
+        {
+            "op": "remove",
+            "path": "/ACL_TABLE/DYNAMIC_ACL_TABLE_BAD",
+            "value": { }
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_failure(output)
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+def dynamic_acl_remove_table_type(duthost):
+    """Remove an ACL Table definition from the duthost
+    As we only have one ACL Table definition on """
+    json_patch = [
+        {
+            "op": "remove",
+            "path": "/ACL_TABLE_TYPE",
+            "value": { }
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+
+
+
+def test_gcu_acl_drop_rule_creation(rand_selected_dut, ptfadapter, setup, dynamic_acl_create_table):
+    dynamic_acl_create_drop_rule_initial(rand_selected_dut, setup)
+    dynamic_acl_verify_packets(setup, ptfadapter, packets = generate_drop_packets(setup), packets_dropped = True)
+    dynamic_acl_verify_packets(setup,
+                               ptfadapter,
+                               packets = generate_drop_packets(setup),
+                               packets_dropped = False,
+                               src_port = setup["unblocked_src_port_indice"])
+
+
+def test_gcu_acl_drop_rule_removal(rand_selected_dut, ptfadapter, setup, dynamic_acl_create_table):
+    dynamic_acl_create_drop_rule_initial(rand_selected_dut, setup)
+    dynamic_acl_remove_drop_rule_initial(rand_selected_dut)
```

Comment:
The test case can be enhanced. For example, we added 3 DROP rules at first, and remove 1 of them, meaning one port is unisolated, and then verify packet ingressed from that port can be forwarded.